src/AzureExtension/Services/DevBox/DevBoxManagementService.cs

```diff
@@ -138,12 +173,24 @@ public async Task<List<DevBoxProjectAndPoolContainer>> GetAllProjectsToPoolsMapp
 
         await Parallel.ForEachAsync(projects.Data!, async (project, token) =>
         {
+            if (await FilterOutProjectAsync(project, developerId))

```

Comment:
>FilterOutProjectAsync [](http://example.com/codeflow?start=22&length=21)

personally, i'd call the function "DeveloperHasWriteAbility" and then put the words "filter out" in a comment above the call to DeveloperHasWriteAbility

Reply:
I like it. Changed.